src/components/viewmodels/right_panel/UserInfoAdminToolsContainerViewModel.tsx

```diff
@@ -0,0 +1,424 @@
+                              );
+                          }
+                        : (child: Room) => {
+                              // Return true if the target member isn't banned and we have sufficient PL to ban
+                              const myMember = child.getMember(cli.credentials.userId || "");
+                              const theirMember = child.getMember(member.userId);
+                              return (
+                                  !!myMember &&
+                                  !!theirMember &&
+                                  theirMember.membership !== KnownMembership.Ban &&
+                                  myMember.powerLevel > theirMember.powerLevel &&
+                                  child.currentState.hasSufficientPowerLevelFor("ban", myMember.powerLevel)
+                              );
+                          },
+                    allLabel: isBanned ? _t("user_info|unban_space_everything") : _t("user_info|ban_space_everything"),
+                    specificLabel: isBanned ? _t("user_info|unban_space_specific") : _t("user_info|ban_space_specific"),
+                    warningMessage: isBanned ? _t("user_info|unban_space_warning") : _t("user_info|kick_space_warning"),
+                },
+                "mx_ConfirmSpaceUserActionDialog_wrapper",
+            ));
+        } else {
+            ({ finished } = Modal.createDialog(ConfirmUserActionDialog, commonProps));
+        }
+
+        const [proceed, reason, rooms = []] = await finished;
+        if (!proceed) {
+            stopUpdating();
+            return;
+        }
+
+        const fn = (roomId: string): Promise<unknown> => {
+            if (isBanned) {
+                return cli.unban(roomId, member.userId);
+            } else {
+                return cli.ban(roomId, member.userId, reason || undefined);
+            }
+        };
+
+        bulkSpaceBehaviour(room, rooms, (room) => fn(room.roomId))
+            .then(
+                () => {
+                    // NO-OP; rely on the m.room.member event coming down else we could
+                    // get out of sync if we force setState here!
+                    logger.info("Ban success");
+                },
+                function (err) {
+                    logger.error("Ban error: " + err);
+                    Modal.createDialog(ErrorDialog, {
+                        title: _t("common|error"),
+                        description: _t("user_info|error_ban_user"),
+                    });
+                },
+            )
+            .finally(() => {
+                stopUpdating();
+            });
+    };
+
+    return {
+        onBanOrUnbanClick,
+        banLabel,
+    };
+};
+
+interface MuteButtonState {
+    isMemberInTheRoom: boolean;
+    muteLabel: string;
+    onMuteButtonClick: () => void;
+}
+
+export const useMuteButtonViewModel = (props: RoomAdminToolsProps): MuteButtonState => {
```

Comment:
Looks like something related to the mute button broke, it's not rendered among the other buttons. See the screenshot failure [here](https://e2e-29808--matrix-react-sdk.netlify.app/#?testId=8dfc0d81b54fac499770-4911a8c051239e1a3ae6).

Reply:
yes indeed the condition to show the button was not correct